orbit-2-savedstate/orbit-2-savedstate_build.gradle.kts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Babylon Partners Limited
+ *  
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("com.android.library")
+    kotlin("android")
+    kotlin("kapt")
+}
+
+apply(from = "$rootDir/gradle/scripts/jacoco-android.gradle.kts")
```

Comment:
I thought this was being automatically included now?

Reply:
yes it is my bad